github/ci/prow/files/jobs/nmstate/kubernetes-nmstate-presubmits.yaml

```diff
@@ -49,7 +49,8 @@ presubmits:
         preset-shared-images: "true"
```

Comment:
You can keep `preset-dind-enabled`, but `preset-docker-mirror` needs to  be removed, so that you don't try to use the docker mirror in the other part of the datacenter.